DigitalOffice/Services/CompanyService/Validators/AddPositionRequestValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.CompanyService.RestRequests;
+
+namespace LT.DigitalOffice.CompanyService.Validators
+{
+    public class AddPositionRequestValidator : AbstractValidator<AddPositionRequest>
+    {
+        public AddPositionRequestValidator()
+        {
+            RuleFor(position => position.Name)
+                    .NotEmpty()
+                    .WithMessage("Position name is empty.")
```

Comment:
Remove it message.

Reply:
Why?

Reply:
https://digitaloffice-group.slack.com/archives/C016H4Z2LGL/p1595663778001900

Reply:
Fixed